src/api/paginateRequest.tsx

```diff
@@ -0,0 +1,16 @@
+import { IPaginateRequest } from "../interfaces/IPaginateRequest";
+
+const paginateRequest = async (pageNumber: number) => {
```

Comment:
Be more specific to name any function. It should tell you what will be retrieved, what is this function doing etc. 
I'm not sure with this api request detailed. But I can suggest something like `getCharacters` `getPaginatedCharacters`...